docs/source/rule-testing.rst

```diff
@@ -47,7 +52,7 @@ All Helpers
 Usage
 -----
 
-To use these helpers in integration test data, replace a specific field with the following syntax::
+To use these helpers in integration test data, replace a specific log field value with the following::
```

Comment:
It's worth defining / explaining if rule tests are integration tests, as the terminology quickly changes